python/GafferCortexUI/StringParameterValueWidget.py

```diff
@@ -55,7 +57,20 @@ def __init__( self, parameterHandler, **kw ) :
 			multiLine = parameterHandler.parameter().userData()["UI"]["multiLine"].value
 
 		if multiLine :
+			
+			# by default the multi-line widget gets auto-expanded in the op interface which works nicely when
```

Comment:
Gaffer's Metadata system is designed so that you can auto-translate metadata from other sources (OSL, Arnold and RSL shader parameters are the main reason) on the fly, rather than explicitly setting them like this. We're already using this system for other metadata on parameters - there are examples here :

https://github.com/ImageEngine/gaffer/blob/master/python/GafferCortexUI/ParameterisedHolderUI.py#L169

We should do the same for this metadata too...